### DISCUSSION TITLE
Allow users to configure an override to the use lima and qemu from a system path

### DISCUSSION THREAD

**Discussion by shri314** [rancher-sandbox/rancher-desktop#3693] [Ideas]
On mac, I see that rancher-desktop bundles its own lima and qemu binaries. This is good to make it work out of the box, however it is limiting as one can get the later versions of qemu and lima via something like `brew install qemu lima`. But, rancher desktop insists to using only its own bundled files. See https://github.com/rancher-sandbox/rancher-desktop/issues/3672 and https://github.com/rancher-sandbox/rancher-desktop/issues/3488.

Why not provide a way to override the configuration to allow use of some other qemu or lima? Like via environment: RD_LIMA_BIN, RD_QEMU_BIN?

This will allow advanced users to not wait a whole rancher desktop release cycle to get the benefits of new lima and qemu releases.

**Comment by jandubois** [comment]
Replied to in https://github.com/rancher-sandbox/rancher-desktop/issues/3694#issuecomment-1369125387; locking discussion to avoid having duplicate threads.
